manager/kubernetes.go

```diff
@@ -48,6 +54,27 @@ func (m *VolumeManager) PVCreate(name, pvName string) (v *longhorn.Volume, err e
 		return nil, err
 	}
 
+	for i := 0; i < KubeStatusPollConut; i++ {
+		v, err = m.ds.GetVolume(name)
+		if err != nil {
+			return nil, err
+		}
+		pvStatus := v.Status.KubernetesStatus.PVStatus
+		if pvStatus == string(apiv1.VolumeAvailable) {
+			break
+		}
+		time.Sleep(KubeStatusPollInterval)
+	}
+
+	v, err = m.ds.GetVolume(name)
```

Comment:
Don't need to get volume again. v is already assigned. Or use a bool `created` for the success flag.